src/SWP/Bundle/TemplateEngineBundle/Model/Container.php

```diff
@@ -77,7 +76,7 @@ class Container implements ContainerInterface, TenantAwareInterface, Timestampab
     protected $widgets;
 
     /**
-     * @var TenantInterface
+     * @var string
```

Comment:
If this is a tenant code, and not a tenant, please change the name of the variable. I don't know why you would to do this, and not create relation, or leave the relation actually.